components/quest-boost/boostCard.tsx

```diff
@@ -35,10 +35,10 @@ const BoostCard: FunctionComponent<BoostCardProps> = ({ boost }) => {
           </div>
           <div className="flex w-full">
             {boost.expiry > Date.now() ? null : (
-              <div className="flex items-center opacity-40">
+              <div className="flex items-center">
                 <div className={styles.issuer}>
-                  <p className="text-white mr-2">Expired</p>
-                  <UnavailableIcon width="24" color="#D32F2F" />
+                  <p className="text-white mr-2">Boost Ended</p>
+                  <CheckIcon width="24" color="#6AFFAF" />
```

Comment:
Imo we should still have something red showing it's not feasible anymore

Reply:
done